<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecipeWidget</class>
 <widget class="QWidget" name="RecipeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RecipeWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="base_vertical_layout">
     <item>
      <widget class="QLabel" name="top_color_accent_label">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(67, 44, 88); </string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="vertical_spacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="recipe_name_label">
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">text-color:rgb(44, 44, 44)</string>
       </property>
       <property name="text">
        <string>Recipe Name</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="vertical_spacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignVCenter">
      <widget class="QLabel" name="recipe_image_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>300</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>10000</width>
         <height>10000</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/resources/images/recipe_1.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="vertical_spacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>50</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="recipe_description_label">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>240</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(103, 103, 103)</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Serve this hearty salad recipe combining red cabbage, blue cheese and glazed walnuts as an accompaniment to anything you can imagine.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="vertical_spacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="click_me_button">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>80</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#FF4081; background-color: #BDBDBD;</string>
       </property>
       <property name="text">
        <string>Click Me</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="vertical_spacer_1">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>click_me_button</sender>
   <signal>clicked()</signal>
   <receiver>RecipeWidget</receiver>
   <slot>OnClickMeButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>774</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>737</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnClickMeButtonClicked()</slot>
 </slots>
</ui>
